benchmarks/ts/src/optimisedmt.ts

```diff
@@ -0,0 +1,348 @@
+/* eslint-disable */
```

Comment:
Let's remove it

Reply:
I'd rather keep it! I added this file as that was our old implementation https://github.com/weijiekoh/optimisedmt - wanted to run benchmarks against it but definitely doesn't make sense to fix their lint mistakes

Reply:
I don't think we need to check in with old implementation in long term. So maybe it makes sense to get rid of it?

Reply:
yea no problem I'll remove! guess it served its purpose now for the pr